src/Symfony/Bundle/WebProfilerBundle/Resources/config/profiler.xml

```diff
@@ -32,6 +34,9 @@
             <argument>%kernel.debug%</argument>
         </service>
 
+        <service id="web_profiler.csp_handler" class="%web_profiler.csp_handler.class%">
+        </service>
```

Comment:
@hhamon I think they are called "empty tags" ... but I absolutely love the "monotag" word :grin: